internal/explorer/helpers.go

```diff
@@ -287,21 +290,58 @@ func (a *App) getNodeData(nodeID string, force bool) (string, error) {
 			}
 			return "", ErrNodeNotFound
 		} else if err != nil {
-			err = a.DeleteRedisKey(fmt.Sprintf("GRID3NODE:%s", nodeID))
-			if err != nil {
-				log.Warn().Err(err).Msg("could not delete key in redis")
+
+			redisData := NodeInfo{}
+			json.Unmarshal([]byte(value), &redisData)
```

Comment:
Unhandled possible error. 

also I  reviewed the `fetchNodeData` method and you are leaking channels by creating them and never closing them
Pleas review this method again

Reply:
The fetchNodeData also does many calls to the node the DMI and hypervisor is never going to change. so u only need to get capacity and version. and I am actually thinking of creating one call that return both so u don't have to make multiple calls to the same node.

Reply:
Will get them once and store it via lru